homeassistant/components/template/light.py

```diff
@@ -67,8 +76,14 @@
     cv.deprecated(CONF_ENTITY_ID),
     vol.Schema(
         {
-            vol.Optional(CONF_COLOR_ACTION): cv.SCRIPT_SCHEMA,
-            vol.Optional(CONF_COLOR_TEMPLATE): cv.template,
```

Comment:
IMHO we can't do this. This will just break about every used light template out there.

Reply:
Should i try to have the 2 naming side by side ?
Or keep what i feel is a confusing name for HS, and add the other stuff around ?

Reply:
That might work, mark that in an exclusive group, so the old still works, the new will work, but setting both throws a configuration error.

Reply:
I've reverted the "deprecation" of the "color" option.
Should we add a warning that it might be at some point, or just leave it like that ?